### ISSUE TITLE
F0038: Date() BIF with (unsupported) Century option segfaults

### ISSUE THREAD

**Issue by rvjansen** [adesutherland/CREXX#286]
say date('w','7688','c')  
ends in:
```
TRACE - RET "CENTURY not allowed as input format"
TRACE - SWAP R2,R24
TRACE - SWAP R3,R25
TRACE - SWAP R5,R26
TRACE - STOI R9
Conversion error occurred

Signal Received - aborting
Deallocating Frames and Registers
Interpreter Finished
Free Symbol Search Trees
➜  tests_functional
```

**Comment by rvjansen** [comment]
currently commented out in ts_date2 test case

**Comment by Peter-Jacob** [comment]
Date raises now a syntax error for wrong input formats: commit 5dd1066a7a8a8d3c549fa263b4fdc318626d154b.
What's the trick to get a trace output, this would be very helpful for debugging

**Comment by rvjansen** [comment]
the trace output: build a debug version of the rxvm ( with -DCMAKE_BUILDTYPE=Debug ) and run that version with -d ;
I always have the release version on my PATH and run the debug with the full path prefixed when needed. Like:
```
~/crexx_debug/interpreter/rxvm -d [your rxbin]
```

**Comment by Peter-Jacob** [comment]
cool, got it running

**Comment by Peter-Jacob** [comment]
Is the problem solved with: commit 5dd1066a7a8a8d3c549fa263b4fdc318626d154b?

**Comment by rvjansen** [comment]
problem (segfault) solved. I'll open a new one for the functionality.
